apps/frontend/public/index.html

```diff
@@ -16,6 +16,46 @@
       property="og:description"
       content="0% interest loans with Sovryn's Zero protocol. Borrow ZUSD or DLLR against your Bitcoin in a fully decentralized system. Increase your earning potential today."
     />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <style>
+      body {
+        font-size: 16px;
+        background: #0D0D0F;
+        color: #ffffff;
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      .sv-container {
+        height: 100%;
+        height: 100vh;
+        width: 100%;
+        display: -webkit-box;
+        display: -ms-flexbox;
+        display: flex;
+        -webkit-box-orient: vertical;
+        -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+        flex-direction: column;
+        -webkit-box-pack: center;
+        -ms-flex-pack: center;
+        justify-content: center;
+        -webkit-box-align: center;
+        -ms-flex-align: center;
+        align-items: center;
+        font-family: 'Roboto', sans-serif;
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      .sv-logo {
+        width: 190px;
+        height: 190px;
+      }
+      .sv-loading-text {
+        margin-top: 10px;
```

Comment:
```suggestion
        margin-top: 18px;
```

increase text size slightly as well please